src/main/java/appeng/core/AppEng.java

```diff
@@ -226,7 +226,7 @@ private void postInit( final FMLPostInitializationEvent event )
 		FMLCommonHandler.instance().registerCrashCallable( new IntegrationCrashEnhancement() );
 
 		CommonHelper.proxy.postInit();
-		AEConfig.instance.save();
+		AEConfig.instance().save();
 
 		NetworkRegistry.INSTANCE.registerGuiHandler( this, GuiBridge.GUI_Handler );
 		NetworkHandler.instance = new NetworkHandler( "AE2" );
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make the enclosing method "static" or remove this set. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)